README.md

```diff
@@ -84,3 +84,8 @@ altered for customization. If some tweaking is required, simply create
 a new configuration file with a ```.local.conf``` suffix and some non
 zero prefix like ```20-user.local.conf```. Files using this suffix are
 on the ```.gitignore``` and not handled as untracked or dirty.
+
+## Contribution
+
+Every help is appreciated, for some guidelines and recommendations check
```

Comment:
Wouldn't "Any help is appreciated" be better?

Side note: since I can't comment on the wiki page, shouldn't we mention the IRC channel, for asking for help / questions